tests/em/static/test_DC_Utils.py

```diff
@@ -75,13 +75,13 @@ def test_io_rhoa(self):
             ],
         ):
             print("\n Testing {} ... ".format(survey_type))
-            survey = utils.gen_DCIPsurvey(
```

Comment:
leave this as the old way of calling to test both the deprecated and new version at the same time

Reply:
Done